src/java/utils/JsonHelper.java

```diff
@@ -0,0 +1,17 @@
+package utils;
+
+import net.sf.json.JSONObject;
+
+/**
+ *
+ * @author markskroba
+ */
+public class JsonHelper {
+    public static JSONObject buildNoneLanguageMap(String label)
+    {
+        JSONObject languageMap = new JSONObject();
+        String[] noneMap = new String[] {label};
```

Comment:
it's so nice when it is easy!

Reply:
Looks good, should end up with `{"none":["some string"]}` as a return.  

As a note, you could even go as far as to return `{"label":{"none":["some string"]}}` which makes sure no one uses anything other than `label` upstream. JSONObjects can be merged/combined/concatenated easily.